torch/csrc/distributed/c10d/TCPStoreLibUvBackend.cpp

```diff
@@ -692,6 +697,12 @@ class LibUVStoreDaemon : public BackgroundThread {
   std::unordered_map<c10::intrusive_ptr<UvHandle>, size_t> keysAwaited_;
   std::unordered_set<c10::intrusive_ptr<UvHandle>> clients_;
   std::unordered_set<c10::intrusive_ptr<UvHandle>> miscellaneousClients_;
+
+  // Queues
+  std::unordered_map<std::string, std::deque<std::vector<uint8_t>>> queues_;
+  std::unordered_map<std::string, std::deque<c10::intrusive_ptr<UvHandle>>>
```

Comment:
looks like this is not being used?

Reply:
good catch -- this was before I switched to using WAIT